flake.nix

```diff
@@ -12,18 +12,26 @@
       nixpkgs,
       utils,
     }:
+    let
+      localOverlay = import ./nix/overlay.nix;
+    in
     utils.lib.eachDefaultSystem (system: rec {
       legacyPackages = import nixpkgs {
         inherit system;
         overlays = [
+          localOverlay
           (final: prev: {
             dev-server = final.callPackage ./nix/dev-server.nix { };
             dev-shell = final.callPackage ./nix/dev-shell.nix { };
```

Comment:
Should these be put in the `localOverlay` as well?

Reply:
These shouldn't be exposed to consumers of the flake, so probably not

Reply:
correct, those are for dev shells. shouldn't be exported

Reply:
We should probably call it `publicOverlay` then and use `localOverlay` for these, or not? Anyways, not so important probably

Reply:
oh, I fell into use of `localOverlay` at work, where we had many overlays floating around. The local one was specific to that repo.